commons/src/main/kotlin/com/simplemobiletools/commons/extensions/Context-storage.kt

```diff
@@ -164,6 +164,10 @@ fun Context.getSAFOnlyDirs(): List<String> {
 }
 
 fun Context.isAccessibleWithSAFSdk30(path: String): Boolean {
+    if(path.startsWith(filesDir.absolutePath)){
```

Comment:
This check helps to exclude files already in the recycle bin. 

 I was thinking of bringing the [`recycleBinPath` defined in the Gallery module](https://github.com/SimpleMobileTools/Simple-Gallery/blob/master/app/src/main/kotlin/com/simplemobiletools/gallery/pro/extensions/Context.kt#L73) to the commons module. Do you think that is a better approach?

Reply:
if you mean the path only and not some huge amount of related code, Im fine with it